apb_devel/writing/reference.adoc

```diff
@@ -0,0 +1,662 @@
+[[apb-devel-writing-reference]]
+= Writing APBs: Reference
+{product-author}
+{product-version]
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+[[apb-devel-writing-ref-overview]]
+== Overview
+
+While the xref:getting_started.adoc#apb-devel-writing-getting-started[Getting Started] topic provides a step by step walkthrough on creating your first APB, this topic provides more in-depth reference material. The fundamental components that make up an APB are explained in further detail to help an experienced APB developer get a better understanding of each individual component within an APB. 
```

Comment:
As suggested with the other topics, I think you should define APB here too, with the first instance. My rationale is if a user landed on this page by search, they would not know that this was defined on the related, more introductory page.